programs/github

```diff
@@ -1,8 +1,38 @@
 local github = dofile("apis/github")
 
-local action = select(1, ...)
+local selectedAction = select(1, ...)
+local passed = { select(2, ...) }
 
-local usage = "Usage: github clone <user>/<repo> [<branch>|<tag>] [<destination>]"
+local subcommands = {
+	clone = {
+		name = 'Clone',
```

Comment:
hrm the name property on these tables isn't really needed anymore, should I kill it?